test/unit/interfaces/hapi.ts

```diff
@@ -16,8 +16,8 @@
 
 import has = require('lodash.has');
 import * as is from 'is';
-var isFunction = (is as {} as {fn: Function}).fn;
-var isObject = is.object;
+const isFunction = (is as {} as {fn: Function}).fn;
```

Comment:
lets use the @types

Reply:
FYI - I just added it:
https://github.com/DefinitelyTyped/DefinitelyTyped/pull/24542

Reply:
Awesome.  Thanks.